components/emoji_picker/emoji_picker_tabs.tsx

```diff
@@ -70,16 +75,22 @@ export default class EmojiPickerTabs extends PureComponent {
                 pickerStyle = {
                     top: this.props.style.top,
                     bottom: this.props.style.bottom,
-                    right: this.props.rightOffset,
+                    right: this.props?.rightOffset,
                 };
             } else {
                 pickerStyle = {...this.props.style};
             }
 
-            pickerStyle.top = pickerStyle.top ? pickerStyle.top + this.props.topOffset : this.props.topOffset;
+            let offset = 0;
+            if (pickerStyle.top) {
+                offset = (this.props.topOffset || 0) + (pickerStyle.top as number);
+            } else {
+                offset = (this.props.topOffset || 0);
+            }
+            pickerStyle.top = offset;
```

Comment:
I think we can improve this part
```suggestion
            pickerStyle.top = this.props.topOffset || 0;
            if (pickerStyle.top && this.props.topOffset) {
                pickerStyle.top = this.props.topOffset + (pickerStyle.top as number);
            }
```